couchdb/src/test/java/org/apache/metamodel/couchdb/CouchDbDataContextTest.java

```diff
@@ -53,7 +53,8 @@ protected void setUp() throws Exception {
         super.setUp();
 
         if (isConfigured()) {
-            httpClient = new StdHttpClient.Builder().host(getHostname()).build();
+            final int timeout = 8 * 1000; // 8 seconds should be more than enough
```

Comment:
`public static final SOCKET_TIMEOUT`?

(I have a great hate for socket timeout settings, because they always seems so arbitrary, and it's hard to find a good number, especially in our case where we really don't know what the use case will be... I guess that's a sign that we at some point should make these user configurable)